### ISSUE TITLE
Improve edit states on cards

### ISSUE THREAD

**Issue by karmatosed** [ParabolInc/parabol#7017]
Note: the team has [a page in Notion](https://www.notion.so/parabol/Task-cards-focus-state-needs-improvement-4e5e48560d9f46249d12bc532d6053e5) on our UX backlog to discuss further

---

The edit states whilst they say 'editing' are too subtle and can get mistaken. In fact, I wasn't sure initially I was editing when first using Parabol.

This shows how subtle it can read:

![2022-08-07 at 13 08](https://user-images.githubusercontent.com/253067/183289854-7636db72-1d91-498c-9069-143851f1e87e.gif)

The visual comes from Chrome, Mac.

My recommendation would be not to go overboard, but look at something simple, border focus, slight shade change - something that denotes a state flow movement and allows the person interacting to feel like they clicked through to an interaction. You don't get that right now.

The serious level is hard to say; it might be that people are more annoyed than anything. You certainly can work it out after a few tries because you can just type. I wonder though how many do actually not see edit or have a frustration point in their experience added through this. Because those points add up, avoiding having one added feels important.

**Comment by acressall** [comment]
I agree, this is really subtle! And whatever we decide to do here could be the same for standup updates. 

Maybe bad idea: replicate the focus state we're using in [solo updates](https://github.com/ParabolInc/parabol/pull/5859)

**Comment by jordanh** [comment]
EE
- Evaluate this design from an a11y perspective
- Design alternates explored, decision on next steps made

AA
- 8 hours

**Comment by enriquesanchez** [comment]
>Maybe bad idea: replicate the focus state we're using in https://github.com/ParabolInc/parabol/pull/5859

@acressall and I looked into this today and we like her idea of using a focus state similar to the hover state we already have.

<img width="349" alt="Captura de Pantalla 2022-09-21 a la(s) 11 39 06 a m" src="https://user-images.githubusercontent.com/3276087/191576599-e800792e-534e-4993-8d59-663169cfff52.png">

Any text area that receives focus should have a focus state, so we think that this state should also be used when reflections and responses receive focus:

<img width="693" alt="Captura de Pantalla 2022-09-21 a la(s) 11 56 44 a m" src="https://user-images.githubusercontent.com/3276087/191576858-dcfae633-1e8d-47b6-b879-d2dafb89a933.png">

<img width="608" alt="Captura de Pantalla 2022-09-21 a la(s) 12 01 25 p m" src="https://user-images.githubusercontent.com/3276087/191576871-d2a54e11-29de-40bc-b147-4e6ea87df280.png">

This makes clear where the user's focus is and also highlights the action of editing a card.

cc. @ackernaut for your review
